release/ray_release/tests/test_db_reporter.py

```diff
@@ -0,0 +1,56 @@
+from ray_release.reporter.db import DBReporter
+
+
+def test_compute_stack_pattern():
+    assert (DBReporter()).compute_crash_pattern(
+        "\n".join(
+            [
+                "haha",
+                "Traceback (most recent call last):",
+                '    File "/tmp/something", line 584',
+                "Exception: yaya45",
+                "hehe",
+            ]
+        )
+    ) == "somethingline Exception: yaya"
+
+
+def test_compute_signature():
+    assert (DBReporter())._compute_signature(
+        [
+            "Traceback (most recent call last):",
+            '   File "/tmp/something", line 584',
+            "Exception: yaya45",
+        ]
+    ) == "somethingline Exception: yaya"
+
+
+def test_compute_stack_trace():
+    trace = [
+        "Traceback (most recent call last):",
+        '   File "/tmp/something", line 584, in run_release_test',
+        "       raise pipeline_exception",
+        "ray_release.exception.JobNoLogsError: Could not obtain logs for the job.",
+    ]
+    error_trace = [
+        "[2023-01-01] ERROR: something is wrong" "Traceback (most recent call last):",
```

Comment:
Should this be 

```suggestion
        "[2023-01-01] ERROR: something is wrong", "Traceback (most recent call last):",
```

?

Reply:
Good catch!